R/new-dataset.R

```diff
@@ -89,7 +89,9 @@ createDataset <- function (name, body, ...) {
 #' @export
 prepareDataForCrunch <- function (data, ...) {
     ## Get all the things
-    message("Processing the data")
+    if (isTRUE(getOption("crunch.show.progress", TRUE))) {
+        message("Processing the data")
+    }
```

Comment:
It might be good to have a separate `progressMessage` function that has the check for show.progress in it.

Reply:
Maybe so, though if we're really thinking about it, I'm skeptical that these messages are really helpful anyway. /shrug

Factoring out a `progressMessage` does help with the coverage game, so I'll invest the 90 seconds on it :)